charts/ztka/Chart.yaml

```diff
@@ -25,5 +25,5 @@ dependencies:
     condition: deploy.contour.enable
 maintainers:
   - name: Paralus Team
-version: 0.0.1-alpha.6
-appVersion: "v0.0.1"
+version: "v0.1.0"
```

Comment:
Let's keep it to `0.1.0` as per the SemVer 2.
https://helm.sh/docs/topics/charts/#charts-and-versioning
https://semver.org/spec/v2.0.0.html#is-v123-a-semantic-version

Reply:
Good catch.

Reply:
Just update `version`, not `appVersion` as it is our application version that we prefixed with `v*`. Sorry for the confusion.